vessel-runtime/src/main/java/com/textnow/android/vessel/VesselCacheImpl.kt

```diff
@@ -0,0 +1,84 @@
+package com.textnow.android.vessel
+
+import java.util.concurrent.ConcurrentHashMap
+import java.util.concurrent.ConcurrentLinkedDeque
+
+/**
+ * [DefaultVesselCache] has no max size and no eviction policy. Data is stored
+ * in memory until the process is killed, or [clear] is called.
+ */
+open class DefaultVesselCache : VesselCache {
```

Comment:
My personal preference is to not include `Vessel` in each of theses - they're not a cache backed by vessel, they're just "caches", but that's more of a personal thing than anything else